src/components/IncorrectCode.jsx

```diff
@@ -0,0 +1,85 @@
+import React, { Fragment } from 'react';
+import PropTypes from 'prop-types';
+import { connect } from 'react-redux';
+import styled from 'styled-components';
+import { Light as SyntaxHighlighter } from 'react-syntax-highlighter';
+import js from 'react-syntax-highlighter/dist/languages/hljs/javascript';
+import Button from './Button';
+import { getJS, getPatterns, getAnswers } from '../selectors';
+import { ATLANTIS, CRIMSON } from '../static/constants/colors';
+
+SyntaxHighlighter.registerLanguage('javascript', js);
+
+const StyledButtonContainer = styled.div`
```

Comment:
Copy-pasted the `StyledButtonContainer`
Can send it to helpers to get rid of repeats